rgd/utility.py

```diff
@@ -7,12 +7,14 @@
 from typing import Generator
 from urllib.request import urlopen
 
+from django.conf import settings
 from django.db.models import fields
 from django.db.models.fields import AutoField
 from django.db.models.fields.files import FieldFile, FileField
 from django.http import QueryDict
 from django.utils.safestring import mark_safe
 from django_filters.rest_framework import DjangoFilterBackend
+from minio_storage.storage import MinioStorage
```

Comment:
@mcovalt This will not work in production, since `minio_storage` is not installed there.

Reply:
Fixed in 38d8e7af114fc4ef39656f369b9195c3f0d6d685